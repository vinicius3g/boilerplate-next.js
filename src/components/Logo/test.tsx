import { render, screen } from '@testing-library/react'

import Logo from '.'

describe('<Logo />', () => {
  it('should render the hading', () => {
    const { container } = render(<Logo />)

    expect(screen.getByRole('heading', { name: /Logo/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
