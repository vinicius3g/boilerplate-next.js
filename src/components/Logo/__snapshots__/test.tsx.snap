// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render the hading 1`] = `
<main
  class=""
>
  <h1>
    Logo
  </h1>
</main>
`;
